src/lib/phdi-transforms/phdi_transforms/geo.py

```diff
@@ -30,7 +30,7 @@ def geocode(client: us_street.Client, address: str) -> GeocodeResult:
     lookup = Lookup(street=address)
     client.send_lookup(lookup)
 
-    if lookup.result:
+    if lookup.result and lookup.result[0].metadata.latitude:
```

Comment:
What's the problem we are trying to solve here? Were we getting back empty results?

Reply:
I did a pretty long local run with the other changes, and then got a response where a seemingly valid response had blank lat/lng and crashed out.

Reply:
I figured it was something like that.